### PR TITLE
Update resource importing documentation for Godot 4.0

### PR DESCRIPTION
**Pull request by Calinou**
- Add best practices for importing images and audio, such as texture size recommendations.

See https://github.com/godotengine/godot-docs/pull/6800 for 3D import documentation updates.

### CODE REVIEW

**Comment by skyace65** [comment]
Importing images is missing a newline at the end

File: tutorials/assets_pipeline/importing_images.rst [hunk 1]
```diff
@@ -40,183 +41,500 @@ Importing textures
...
+    For technical reasons, the editor must be restarted after changing an import
+    type in the Import dock.
 
 Detect 3D
-~~~~~~~~~
+^^^^^^^^^
+
+The default import options (no mipmaps and **Lossless** compression) are suited
+for 2D, but are not ideal for most 3D projects. **Detect 3D** makes Godot aware
+of when a texture is used in a 3D (such as a texture in a
```
Comment (skyace65) [inline tutorials/assets_pipeline/importing_images.rst:109]:
```suggestion
of when a texture is used in a 3D scene (such as a texture in a
```

File: tutorials/assets_pipeline/importing_images.rst [hunk 2]
```diff
@@ -40,183 +41,500 @@ Importing textures
...
++---------------+---------------------+---------------------+---------------------+---------------------+---------------------+
+| **512×512**   | |good| 1.33 MiB     | |good| 1.33 MiB     | |good| 341 KiB      | |good| 1.33 MiB     | |good| 341 KiB      |
++---------------+---------------------+---------------------+---------------------+---------------------+---------------------+
+| **1024×1024** | |regular| 5.33 MiB  | |regular| 5.33 MiB  | |good| 1.33 MiB     | |regular| 5.33 MiB  | |good| 1.33 MiB     |
++---------------+---------------------+---------------------+---------------------+---------------------+---------------------+
+| **2048×2048** | |bad| 21.33 MiB     | |bad| 21.33 MiB     | |regular| 5.33 MiB  | |bad| 21.33 MiB     | |regular| 5.33 MiB  |
++---------------+---------------------+---------------------+---------------------+---------------------+---------------------+
+| **4096×4096** | |bad| 85.33 MiB     | |bad| 85.33 MiB     | |bad| 21.33 MiB     | |bad| 85.33 MiB     | |bad| 21.33 MiB     |
++---------------+---------------------+---------------------+---------------------+---------------------+---------------------+
```
Comment (skyace65) [inline tutorials/assets_pipeline/importing_images.rst:220]: Not sure if we should be saying "good" or "bad," might be a better idea to just give the size and let people make their own decisions. The way this is made makes it seem like you should never use 4K textures.

**Review by skyace65 — CHANGES_REQUESTED** [review]
Looks good! Just needs a few fixes.

File: tutorials/assets_pipeline/importing_images.rst (hunk 2 above)
Comment (Calinou) [reply tutorials/assets_pipeline/importing_images.rst:220]: The "good" or "bad" badges are mostly for visual grepping – they're not visible text on the page.

File: tutorials/assets_pipeline/import_process.rst [hunk 3]
```diff
@@ -3,28 +3,23 @@
...
+Importing assets in Godot
+-------------------------
 
-Previously, importing assets in Godot 2.x required manual maintenance
-of a separate directory with source assets. Without doing this, it was
-impossible to specify how to convert and change import flags for
-textures, audio files, scenes, etc.
-
-In Godot 3.0+, we use a more modern approach to importing: Simply drop
+In Godot 3.0we use a more modern approach to importing: Simply drop
```
[the remark this answers is not in the document]
Comment (Calinou) [reply tutorials/assets_pipeline/import_process.rst:9]: I've reworded this paragraph entirely, please review the updated version.

File: tutorials/assets_pipeline/import_process.rst [hunk 4]
```diff
@@ -66,48 +74,77 @@ automatic reimport of it, applying the preset configured for that specific
...
 
-Importing will add an extra ``<asset>.import`` file, containing the import
-configuration. Make sure to commit these to your version control system!
+    $ ls
+    example.png
+    example.png.import
+    project.godot
 
-.. image:: img/asset_workflow4.png
+Additionally, extra assets will be preset in the hidden
```
Comment (skyace65) [inline tutorials/assets_pipeline/import_process.rst:92]:
```suggestion
Additionally, extra assets will be present in the hidden
```

**Review by skyace65 — CHANGES_REQUESTED** [review]
Looked over the stuff you changed and it looks good now. Just found 1 more issue.

**Review by skyace65 — APPROVED** [review]

**Review by mhilbrunner — APPROVED** [review]

**Comment by mhilbrunner** [comment]
Awesome, thanks! 🎉 Merged.

**Pull request merged**